experiments/compositions/composition/config/expanders/expander_versions.yaml

```diff
@@ -30,7 +31,7 @@ spec:
 apiVersion: composition.google.com/v1alpha1
 kind: ExpanderVersion
 metadata:
-  name: gjinja2
+  name: jinja2
   namespace: system
 spec:
   type: grpc
```

Comment:
does this mean that for grpc we name it to the default jinja2, and for type job, we call it pjinja2? 
nit: Is there a reason why it is called pjinja?

Reply:
Default is grpc. Yes. 

pjinja2 -> pod jinja2 :) 

planning to rename it to jinja2-job to be explicit.

Reply:
Thanks for the explanation!